ui/src/app/pages/workspace/workspace-share.tsx

```diff
@@ -261,20 +261,21 @@ export const WorkspaceShare = fp.flow(withUserProfile())(class extends React.Com
   }
 
   userSearch(value: string): void {
+    const {accessTierShortName} = this.props.workspace;
+    const {searchTerm, userRoles} = this.state;
     this.setState({autocompleteLoading: true, autocompleteUsers: [], searchTerm: value});
     if (!value.trim()) {
       this.setState({autocompleteLoading: false, dropDown: false});
       return;
     }
-    const searchTerm = this.state.searchTerm;
-    userApi().user(this.state.searchTerm)
```

Comment:
this is the only use of this endpoint.  We can remove the endpoint after this is released.